### DISCUSSION TITLE
Can't Trade in Bitget

### DISCUSSION THREAD

**Discussion by shaman1oo864** [soranoo/TradingView-Free-Webhook-Alerts#53] [Q&A]
Hello. I think it is a very good idea to send an e-mail - SMS. However I have a problem. Everything is fine in the test, using https://webhook.site/ I see the body of the hook ok. but there is no action in the exchange. no opening of orders, no unsuccessful signals. Log program ok. I tried Binance and BitGet. I read the instructions and watched the video many times. I did it correctly.

Tell me, how can I make a webhook?

**Comment by soranoo** [comment]
Do you mean you want to make a webhook server?

**Comment by shaman1oo864** [comment]
No, I no have signal to Bitget. I input my string to config (https://api.bitget.com/api/strategy/v1/trading-view/callback?token=68e2b02e40ab9df.........f2) and fill 
{
"action":"{{strategy.order.action}}",
"contracts":"{{strategy.order.contracts}}",
"positionSize":"{{strategy.position_size}}",
"price":"{{close}}",
"symbol":"{{ticker}}",
"time":"{{timenow}}"
}
No reaction BitGet, 
but OK on https://webhook.site/aa2f647a-187d-485a-81f4-703bad61a9a7

**Reply by soranoo** [reply]
Be careful. You have leaked your API token. Please renew the token.

**Comment by shaman1oo864** [comment]
Success     Workflow executed in 6 183 ms
details     Total Duration:   6 183 ms
Compute Time:     5 778 ms
Execution Start:     2023-10-18T20:21:07.107Z
Execution End:      2023-10-18T20:21:13.290Z

Maybe the problem is in the BitGet settings? Cross / Isolate or hedging / open orders???

**Comment by shaman1oo864** [comment]
{
"action":"buy",
"contracts":"8.7",
"positionSize":"2.9",
"price":"0.48898",
"symbol":"XRPUSDT.P",
"time":"2023-10-18T20:21:04Z"
}

**Comment by shaman1oo864** [comment]
I see in page BitGet "symbol": "XRPUSDT_UMCBL" (api description) , but here "symbol":"XRPUSDT.P" ?

**Reply by soranoo** [reply]
Which symbol did you use on TV? Are you choosing the wrong symbol?

**Reply by shaman1oo864** [reply]
I used both characters. No effect.

**Comment by shaman1oo864** [comment]
Or very long time ( for example : Workflow executed in 7 108 ms) ?

**Comment by shaman1oo864** [comment]
And I use 3commas (directly signal TV) for open order via email-sms . All OK.

**Comment by shaman1oo864** [comment]
I send message to BitGet. Now I read :

"For now the trading view webhook can only share you alert, it can not help you to trade so if you want it to help you to execute trades or orders it can not happen for now. It is still under development."

This horse is slow, shoot him, Joe.

**Comment by shaman1oo864** [comment]
And add BitGet:

"We will inform you in this ticket also you can keep an eye on our announcement center, we will release announcement once the TV webhook is online."

**Comment by soranoo** [comment]
Ok, short answer: BitGet's TradingView Trading Webhook is still under **development**.
